httptransport/server.go

```diff
@@ -168,52 +161,20 @@ func (t *Server) configureComboMode(ctx context.Context) error {
 func (t *Server) configureIndexerMode(_ context.Context) error {
 	// requires only indexer service
 	if t.indexer == nil {
-		return clairerror.ErrNotInitialized{"IndexerMode requires an indexer service"}
+		return clairerror.ErrNotInitialized{Msg: "IndexerMode requires an indexer service"}
 	}
 
-	// affected manifest handler register
-	affectedH := intromw.Handler(
-		othttp.NewHandler(
-			LoggingHandler(AffectedManifestHandler(t.indexer)),
-			AffectedManifestAPIPath,
-			t.traceOpt,
-		),
-		AffectedManifestAPIPath,
-	)
-	t.Handle(AffectedManifestAPIPath, othttp.WithRouteTag(AffectedManifestAPIPath, affectedH))
-
-	// index handler register
-	indexH := intromw.Handler(
-		othttp.NewHandler(
-			LoggingHandler(IndexHandler(t.indexer)),
-			IndexAPIPath,
-			t.traceOpt,
-		),
-		IndexAPIPath,
-	)
-	t.Handle(IndexAPIPath, othttp.WithRouteTag(IndexAPIPath, indexH))
-
-	// index report handler register
-	indexReportH := intromw.Handler(
-		othttp.NewHandler(
-			LoggingHandler(IndexReportHandler(t.indexer)),
-			IndexReportAPIPath,
-			t.traceOpt,
-		),
-		IndexReportAPIPath,
-	)
-	t.Handle(IndexReportAPIPath, othttp.WithRouteTag(IndexReportAPIPath, indexReportH))
-
-	// index state handler register
-	stateH := intromw.Handler(
-		othttp.NewHandler(
-			LoggingHandler(IndexStateHandler(t.indexer)),
-			IndexStateAPIPath,
-			t.traceOpt,
-		),
-		IndexStateAPIPath,
-	)
-	t.Handle(IndexStateAPIPath, othttp.WithRouteTag(IndexStateAPIPath, stateH))
+	t.Handle(AffectedManifestAPIPath,
+		intromw.InstrumentedHandler(AffectedManifestAPIPath, t.traceOpt, AffectedManifestHandler(t.indexer)))
+
+	t.Handle(IndexAPIPath,
+		intromw.InstrumentedHandler(IndexAPIPath, t.traceOpt, IndexHandler(t.indexer)))
+
+	t.Handle(IndexReportAPIPath,
+		intromw.InstrumentedHandler(IndexReportAPIPath+"GET", t.traceOpt, IndexReportHandler(t.indexer)))
```

Comment:
Why does this have an extra string tacked on?

Reply:
```
	IndexAPIPath            = indexerRoot + apiRoot + "index_report"
	IndexReportAPIPath      = indexerRoot + apiRoot + "index_report/"
```

when both those paths are normalized to a Prom metric, if we don't tack on an extra string to differentiate, they will have the same metric name, and registration of the metric will panic.